### PR TITLE
Feature/constants

### PR DESCRIPTION
**Pull request by sfregolina**
I created a constants file and populated it with variables. I also moved all my scss files to their relevant folder.

### CODE REVIEW

File: src/sass/common/constants.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+$fontColorWhite: #FFF;
+
+$backgroundColorWhite: #FFF;
+$backgroungColorBlack: #000;
+
+$borderWidth: 1px solid;
+
+$fullWidthComponent: 100%;
+
+$fontBold: 700;
```
Comment (Flamov) [inline src/sass/common/constants.scss]: Empty line at the end of the file please!

File: src/sass/common/mixins.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@mixin display-flex() {
+@mixin display-flex () {
```
Comment (Flamov) [inline src/sass/common/mixins.scss]: You don't need parentheses (`()`) for mixins that have no arguments!

File: src/sass/style.scss [hunk 3]
```diff
@@ -1,10 +1,10 @@
-@import "mixins", "navigation", "main", "footer";
+@import "common/mixins", "common/constants", "components/navigation", "components/main", "components/footer";
```
Comment (Flamov) [inline src/sass/style.scss]: For a long amount of imports (maybe 4 or more?) I'd suggest spitting them into separate lines for clarity:

```suggestion
@import "common/mixins";
@import "common/constants";
@import "components/navigation";
@import "components/main";
@import "components/footer";
```

File: src/sass/common/constants.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+$fontRoboto: "Roboto", sans-serif;
+$fontMontserrat: "Montserrat", sans-serif;
+
+$fontColorBlack: #000;
+$fontColorWhite: #FFF;
+
+$backgroundColorWhite: #FFF;
+$backgroungColorBlack: #000;
+
+$borderWidth: 1px solid;
```
Comment (Flamov) [inline src/sass/common/constants.scss]: The constants are great (good job!) but for ones like these I'd ask a couple of questions before turning it into a constant:

* Will this be changed in the future?
* Is it part of a design that can be changed easily (e.g. can we change `1px` to `3px` and will it still look okay for all rules using this constant?)

File: src/sass/common/mixins.scss (hunk 2 above)
Comment (sfregolina) [reply src/sass/common/mixins.scss]: Sorry, I haven't realised I could remove the brackets from mixins.

**Pull request merged**